open-sphere-base/core/src/main/java/io/opensphere/core/viewer/impl/DynamicViewer.java

```diff
@@ -60,6 +60,20 @@
      */
     double getHeading();
 
+    /**
```

Comment:
I don't really think a fly-to member belongs in the viewer, as it's a property that's specific to animation, not the viewer.